pkg/apis/networking/types.go

```diff
@@ -144,15 +144,15 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString
 }
 
-// IPBlock describes a particular CIDR (Ex. "192.168.1.1/24","2001:db9::/64") that is allowed
+// IPBlock describes a particular CIDR (Ex. "192.168.1.0/24","2001:db9::/64") that is allowed
```

Comment:
can we also add the fact that `IPBlock` generally applies to endpoints?
some folks have tried to use this to target non-pod services, which can lead
to alot of confusion